### ISSUE TITLE
App crashes when doing first test

### ISSUE THREAD

**Issue by Wraeyth** [A9T9/Speed-Test-Loggger#10]
I'm using http://ipv4.download.thinkbroadband.com/1GB.zip as my download, and before the first run completes the application crashes.

When I click Debug, the detail I get is:

"An unhandled Microsoft >NET Frarmework exception occurred in speedtestloggger.exe [49220]"

**Comment by Wraeyth** [comment]
Using a 512MB file still crashed, but with a 200MB file - no crashes.

**Comment by bleomycin** [comment]
Same issue here, win 10 64bit. 1GB file causes crash, 100MB is fine.

**Comment by thinze** [comment]
Yes, I have program crashes too. With 512MB file the crashes are often as the 200 MB file.

I research for possible reasons and found this error log in /AppData/local/temp/
<ProblemSignatures>
	<EventType>CLR20r3</EventType>
	<Parameter0>speedtestloggger.exe</Parameter0>
	<Parameter1>1.0.0.0</Parameter1>
	<Parameter2>54cea3ca</Parameter2>
	<Parameter3>System</Parameter3>
	<Parameter4>4.7.2117.0</Parameter4>
	<Parameter5>59cf51b3</Parameter5>
	<Parameter6>4089</Parameter6>
	<Parameter7>157</Parameter7>
	<Parameter8>System.OutOfMemoryException</Parameter8>
</ProblemSignatures>

I dont know what the reason for large memory usage is.
